src/pymodule/conversions.rs

```diff
@@ -14,7 +14,7 @@ use pyo3::PyErr;
 impl From<PConvertError> for PyErr {
     fn from(err: PConvertError) -> PyErr {
         match err {
-            PConvertError::ArgumentError(err) => PyAttributeError::new_err(err.to_string()),
```

Comment:
Not related to tests -> different PR. You can make the clippy stuff from all PRs in a single one.